pages/common/rails-generate.md

```diff
@@ -1,19 +1,24 @@
 # rails generate
 
 > Generate new Rails templates in an existing project.
+> More information: <https://guides.rubyonrails.org/command_line.html#rails-generate>.
 
 - List all available generators:
 
 `rails generate`
 
-- Generate a new model:
+- Generate a new model named Post with attributes title and body:
```

Comment:
Maybe we should have gradually increased in complexity by adding another example which adds the attributes. Since there is space for more examples, it could be helpful in easing the users trying to grok the command.

Reply:
Ah, that's a good idea.

I'm not a Ruby programmer, so I didn't pick up on this :P